docs/api/rtos/EventFlags.md

```diff
@@ -10,33 +10,4 @@ The EventFlags class provides a mechanism for setting and waiting for specific c
 
 ### EventFlags example
 
-Below is an example of EventFlags usage, where one thread is generating events every 1s, and the second thread is waiting for the events and executing some action.
```

Comment:
Query: Is this deleted description still relevant? If so, could we please keep it?

Reply:
Since I've slightly updated the example code, the description should also be updated. Do you prefer to have it here, or should I rather edit the new README file https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/EventFlags/README.md ?

Reply:
If the example is self-explanatory enough that it doesn't need setup, feel free to edit it only in the README file. If the example needs some context, please edit it here.

Reply:
Good question 👍

Reply:
README updated in a new PR https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/8 since the last one got merged already.